stackage_snapshot.json

```diff
@@ -1,6 +1,6 @@
 {
-  "__GENERATED_FILE_DO_NOT_MODIFY_MANUALLY": -424426183,
-  "all-cabal-hashes": "https://raw.githubusercontent.com/commercialhaskell/all-cabal-hashes/4993fc769cca0c69efec9bf28899e15fe3ed2d99",
+  "__GENERATED_FILE_DO_NOT_MODIFY_MANUALLY": -2052748493,
```

Comment:
I don't recall why this updated. I know i needed to add `array` as an exposed dep, but I see the diff is only updating hashes... I'll check

Reply:
Ah right, I had run the stackage script to see if any changes were needed to add `array` to stackage. It didn't, but some of the packages we rely on have had revisions (metadata only changes). This change is safe, but not strictly needed. I had figured it can't hurt to include, but I can remove if it we're iffy about this.

Reply:
Yeah let's just remove it.